packages/storybook/stories/snippets/VideoEffectsSnippet.snippet.tsx

```diff
@@ -0,0 +1,69 @@
+import { AzureCommunicationTokenCredential, CommunicationUserIdentifier } from '@azure/communication-common';
+import { AzureCommunicationCallAdapterOptions } from '@azure/communication-react';
+import { onResolveVideoEffectDependencyLazy } from '@azure/communication-react';
+import { CallComposite, CallCompositeOptions, useAzureCommunicationCallAdapter } from '@azure/communication-react';
+import { PartialTheme, Stack, Theme } from '@fluentui/react';
+import React, { useMemo } from 'react';
+
+export type ContainerProps = {
+  userId: CommunicationUserIdentifier;
+  token: string;
+  groupId: string;
+  formFactor?: 'desktop' | 'mobile';
```

Comment:
Is formFactor and fluentTheme important for this code snippet? If not let's remove them all togehter

Reply:
Including options - we should provide minimal code to get things work to reduce distraction